pkg/skaffold/schema/versions_test.go

```diff
@@ -668,40 +675,38 @@ func withLogsPrefix(prefix string) func(*latest_v1.SkaffoldConfig) {
 }
 
 func TestUpgradeToNextVersion(t *testing.T) {
-	for i, schemaVersion := range SchemaVersions[0 : len(SchemaVersions)-2] {
-		from := schemaVersion
-		to := SchemaVersions[i+1]
-		description := fmt.Sprintf("Upgrade from %s to %s", from.APIVersion, to.APIVersion)
+	for _, versions := range []Versions{SchemaVersionsV1, SchemaVersionsV2} {
+		for i, schemaVersion := range versions[0 : len(versions)-2] {
+			from := schemaVersion
+			to := SchemaVersionsV1[i+1]
```

Comment:
```suggestion
			to := versions[i+1]
```
i think this should be changed so that it doesn't break when testing upgrades for V2 versions

Reply:
good catch!